src/client/components/MyProfile/ProfilePage/ProfileComponent.js

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState, useContext } from 'react';
+import { handlerUserContext } from '../../../App';
```

Comment:
@pdsorensen  Could you guide me to handle these failures? regarding the dependency cycle via my components.

Reply:
You should just remove this line. It's not being used right?

Reply:
https://github.com/HackYourFuture-CPH/rate-my-cv/pull/159#discussion_r575149357
@pdsorensen As we agreed previously , I made another component now with ```context``` tried to avoid drilling the state from ```app``` to  ```<Profile>``` then to ```<ProfileComponent>```  Now I go and do the drilling if you say so?

Reply:
@arya56 that still makes sense. Try to draw the depedency graph, and you will observe the problem. Give it 2 mins before reading the solution, and maybe get inspired from this: https://reactjs.org/docs/context.html

Reply:
(the solution i think is.................... to put the context into another file)